templates/global/quantity-input.php

```diff
@@ -41,8 +41,8 @@ class="<?php echo esc_attr( join( ' ', (array) $classes ) ); ?>"
 			title="<?php echo esc_attr_x( 'Qty', 'Product quantity input tooltip', 'woocommerce' ); ?>"
 			size="4"
 			inputmode="<?php echo esc_attr( $inputmode ); ?>"
-			<?php if ( ! empty( $labelledby ) ) { ?>
-			aria-labelledby="<?php echo esc_attr( $labelledby ); ?>" />
+			<?php if ( ! empty( $label ) ) { ?>
+			aria-label="<?php echo esc_attr( $label ); ?>" />
```

Comment:
This input already has a label (just above here), is there a reason we need to add the aria-label too? I see the text is different, the aria-label is "Product Name Quantity". What if instead, the text in `label` was changed to use the product name if it's available? That's effectively what's happening here, if the product name exists the screen reader reads "Product Name Quantity", and if it's not it just reads "Quantity".

Reply:
Not particularly, aria-labelledby was being used incorrectly so I just thought to correct it, but I agree it's a bit redundent it may indeed be better to change the existing label.